src/Kafka/KafkaInstanceDrawer/KafkaConnectionTab.tsx

```diff
@@ -56,6 +62,69 @@ export const KafkaConnectionTab: FunctionComponent<KafkaConnectionTabProps> = ({
           </ClipboardCopy>
         )}
       </TextContent>
+      <TextContent className="pf-u-pb-sm">
+        <Text component={TextVariants.h3} className="pf-u-mt-lg">
+          {t("kafka:connection_tab.rest_api")}{" "}
+          <Popover
+            aria-label={t("kafka:connection_tab.rest_api_popover_label")}
+            bodyContent={
+              <div>
+                <div>
+                  {
+                    <Trans
+                      i18nKey={
+                        "kafka:connection_tab.rest_api_popover_content_1"
+                      }
+                      components={{
+                        value: <Link to={linkToAPIDocumentation}></Link>,
+                      }}
+                    />
+                  }
+                </div>
+                <div>
+                  {
+                    <Trans
+                      i18nKey={
+                        "kafka:connection_tab.rest_api_popover_content_2"
+                      }
+                      components={{
+                        value: <Link to={linkToLearnMore}></Link>,
+                      }}
+                    />
+                  }
+                </div>
+              </div>
+            }
+          >
+            <Button
+              variant={ButtonVariant.plain}
+              aria-label={t("kafka:connection_tab.rest_api_help_button_label")}
+            >
+              <HelpIcon />
+            </Button>
+          </Popover>
+        </Text>
+        <Text component={TextVariants.small}>
+          {
+            <Trans
+              i18nKey={"kafka:connection_tab.rest_api_url_description"}
+              components={{
+                value: <Link to={linkToAPIDocumentation}></Link>,
+              }}
+            />
+          }
+        </Text>
+        {isKafkaPending ? (
+          <Skeleton fontSize="2xl" />
+        ) : (
+          <ClipboardCopy
+            data-testid="drawerStreams-copyRestApiURL"
+            textAriaLabel={t("kafka:connection_tab.rest_api")}
+          >
+            {restAPIUrl}
+          </ClipboardCopy>
```

Comment:
I'm just now noticing that we have one clipboard copy that is gray and one that is white. My guess is that all of these should be gray (i.e. read-only), but I can confirm that with stakeholders.
 
![image](https://user-images.githubusercontent.com/21063328/162337230-c19df7e5-7716-4312-8cab-689ae0a70898.png)
![image](https://user-images.githubusercontent.com/21063328/162337248-eb51329d-e16c-41cd-ab1a-bae4dd03629d.png)

Reply:
Following @wtrocki's feedback, let's make all clipboard copy elements read-only (i.e. gray background).